L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -18,10 +20,8 @@ void PrimitiveConversion::configure(
     bool duplicateTheta, bool fixZonePhi, bool useNewZones, bool fixME11Edges,
     bool bugME11Dupes
 ) {
-  if (not(tp_geom != nullptr))
-    { edm::LogError("L1T") << "tp_geom = " << tp_geom; return; }
-  if (not(lut != nullptr))
-    { edm::LogError("L1T") << "lut = " << lut; return; }
+  assert(tp_geom != nullptr);
```

Comment:
No asserts.

Reply:
This belongs to coding errors, not input/runtime dependent. I think assert should be allowed.